src/aws_encryption_sdk/streaming_client.py

```diff
@@ -751,9 +753,21 @@ def _read_header(self):
         )
         return header, header_auth
 
+    @property
+    def body_start(self):
+        """Log deprecation warning when body_start is accessed."""
+        _LOGGER.warning("StreamDecryptor.body_start is deprecated and will be removed in 1.4.0")
```

Comment:
Would adding this deprecation message be released through a minor rev or a patch rev?

Reply:
Per our versioning statement, the warnings should be added in a `Z` rev and the removal should happen in a `Y` rev.

https://github.com/aws/aws-encryption-sdk-python/blob/master/VERSIONING.rst